backend/README.md

```diff
@@ -1,6 +1,6 @@
 # Hanko backend
 
-The Hanko backend provides public and private APIs that offer a set of functions to create a modern login and registration experience for your users. Hanko backend works well with [`<hanko-auth>`](../hanko-js/README.md), a web component that can easily be integrated in any web app and provides ready-to-use login and registration UI flows.
+The Hanko backend provides public and private APIs that offer a set of functions to create a modern login and registration experience for your users. Hanko backend works well with [hanko-js](../hanko-js), a web component that can easily be integrated in any web app and provides ready-to-use login and registration UI flows.
```

Comment:
hanko-js is not a web component, but among other things you can use hanko-js to register a web component.

Reply:
A small suggestion here would be not to put the web component in the center, but to say that the backend consists of an API for passkeys/webauthn, passwords and passcodes, and manages JWT's.

Reply:
Hmm, I read through it again. About the upcoming features we say that backoff mechanisms will come, but at the bottom of the text it says:

> The Hanko service does not implement rate limiting in any way. In production systems, you want to hide the Hanko service behind a proxy or gateway (e.g. kong, traeffik) that implements rate limiting.
 
And exponential backoff is is a rate limiting algorithm.

Reply:
Thanks for your feedback. How about that:

```suggestion
Hanko backend provides public and private APIs that offer a set of functions to create a modern login and registration experience for your users. Its core features are an API for passkeys (WebAuthn), passwords, and passcodes, as well as JWT management.

Hanko backend can be used on its own or in combination with [hanko-js](../hanko-js), a powerful frontend library that contains polished and customizable UI flows for password-based and passwordless user authentication that can be easily integrated into any web app with as little as two lines of code.
```

Reply:
> Hmm, I read through it again. About the upcoming features we say that backoff mechanisms will come, but at the bottom of the text it says:
> 
> > The Hanko service does not implement rate limiting in any way. In production systems, you want to hide the Hanko service behind a proxy or gateway (e.g. kong, traeffik) that implements rate limiting.
> 
> And exponential backoff is is a rate limiting algorithm. I would suggest either taking it out of the list of upcoming features or being more precise in the text, or the other way around, depending on how we want it to work later.

Addressed this here https://github.com/teamhanko/hanko/pull/110/commits/91fa76d4805ee1670b8ef6f7583faf7975ec5f8b